bin/sirius-script-rabpm-mngr.py

```diff
@@ -31,8 +32,10 @@ def _parse_agent_env(self, output: bytes):
         return result
 
     def execute_ssh(self, hostname, *command):
-        subprocess.run(["ssh", "-i", "~/.ssh/id_ed25519_rabpm",
-                       f"lnls-bpm@{hostname}", *command])
+        self.procs.append(subprocess.Popen(
+            ["ssh", "-i", "~/.ssh/id_ed25519_rabpm",
+                f"lnls-bpm@{hostname}", *command],
+            stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL))
```

Comment:
It is missing the rationale behind the usage of DEVNULL in the commit message. You mention it omits the output, but why has this chosen to be done now? Why does this relate to this patch at all?

Remember you should make it possible for a future hypothetical developer to understand why this was chosen to be done this way by reading the commit message.